pages/who_we_serve.js

```diff
@@ -30,90 +37,125 @@ const mentorItems = [
-              analyticsEventLabel="Meetup.com Locations Link"
-              theme="secondary"
+            <div className={classNames(styles.centeredText, styles.topMargin)}>
+              <LinkButton href="/join" theme="secondary">
+                Become A Member
+              </LinkButton>
+            </div>
+          </div>,
+        ]}
+      />
+
+      <Content
+        columns={[
+          <TrackVisibility offset={VISIBILITY_OFFSET}>
+            <ImageCard
+              alt="Two developers collaborting over some code."
+              imageSource={`${s3}redesign/images/paired_programming.jpg`}
+              isImageFirst
             >
-              See Locations
-            </LinkButton>
-          </ImageCard>
-        </TrackVisibility>,
-      ]}
-    />
-
-    <JoinSection />
-  </>
-);
+              <p className={styles.centeredText}>
+                Currently 1,000,000+ <br /> software development jobs available in the United
+                States.
+              </p>
+            </ImageCard>
+          </TrackVisibility>,
+        ]}
+      />
+
+      <Content
+        title="Our Commitment To You"
+        theme="gray"
+        columns={[
+          <p className={styles.justifyAlign}>
+            Whether you are looking to change careers or starting a new one in the tech industry, we
+            are here to help you succeed by providing:
+          </p>,
+          <div className={styles.badgeGroupings}>
+            {mentorItems.map(item => (
+              <Badge
+                key={item.label}
+                icon={item.icon}
+                label={item.label}
+                className={styles.badge}
+              />
+            ))}
+          </div>,
+        ]}
+      />
+
+      <Content
+        columns={[
+          <TrackVisibility offset={VISIBILITY_OFFSET}>
+            <ImageCard
+              alt="Room full of people chatting at an Operation Code meetup in New York City."
+              imageSource={`${s3}redesign/images/chatting_at_meetup.jpg`}
+            >
+              <p className={styles.centeredText}>
+                We have meetup chapters all around the United States!
+              </p>
+
+              <LinkButton
+                href="https://www.meetup.com/pro/operationcode"
+                analyticsEventLabel="Meetup.com Locations Link"
+                theme="secondary"
+              >
+                See Locations
+              </LinkButton>
+            </ImageCard>
+          </TrackVisibility>,
+        ]}
+      />
+
+      <JoinSection />
+    </>
+  );
+}
+
+WhoWeServe.getInitialProps = getMemberCount;
+
+async function getMemberCount() {
+  const response = await axios.get(slackApiUrl.concat(slackConversationsMembersEndpoint), {
+    params: {
+      token: process.env.SLACK_API_TOKEN,
+      channel: slackGeneralChannelId,
+    },
+  });
+
+  return {
+    memberCount: response.data.ok ? response.data.members.length : null,
```

Comment:
If the API doesn't work, `response` won't have the property data nor data.ok which would yield an error.

```suggestion
    memberCount: response && response.data && response.data.ok ? response.data.members.length : null,
```

Reply:
or use `lodash/get`

```suggestion
    memberCount: get(response, 'data.members.length', null)
```